### DISCUSSION TITLE
can I both use babel v6 and v7 in my project?

### DISCUSSION THREAD

**Discussion by touchpillow** [babel/babel#15054] [Q&A] [answered]
My project was created several years ago, and it's use babel 6.x.

Now, I need to import a plugin that depend on babel 7.x , and I try to use it, and didn't find problems yet, but I know it's a dangerous action to keep two different mainly babel version in one project.

And it will make a big influence if I upgrade the babel version from 6.x to 7.x, I guess I have no time to do it. So can I both use babel 6 and 7 in my project, and can anyone told me the risk of it?

**Comment by nicolo-ribaudo** [comment, marked as answer]
It's "without warranty": it may work, but if it breaks the only option you have is to fully migrate to Babel 7.

**Reply by touchpillow** [reply]
yes, I know that, so the best way is upgrade to babel 7 totally, the risk is lower even through with more points
